ReactWindows/ReactNative/Modules/Network/NetworkingModule.cs

```diff
@@ -123,7 +124,26 @@ private RCTDeviceEventEmitter EventEmitter
                         return;
                     }
 
-                    throw new NotImplementedException("URI upload is not supported");
+                    var streamTask = 
```

Comment:
This is fine, I just don't like the idea that we're allocating two additional closures (the Func arguments to Select and SelectMany) when we could do just as easily without.  Also, I think C# developers tend to understand async/await more readily that LINQ operator overloading.